### ISSUE TITLE
%rapidweaverAssetPath% different in S3

### ISSUE THREAD

**Issue by jrondorf** [yourhead/s3#335]
I am experiencing differences in the %rapidweaverAssetPath% template between S2 and S3.

In S2.7, I am having a Stacks page in a subdirectory like /foldername/
%rapidweaverAssetPath% is returning there **../resources**

In S3, I am getting following return value: **resources**

**Comment by designdisorder** [comment]
It obviously not respecting the relative address scheme, which I assume is the setting you are using in RW

**Comment by jrondorf** [comment]
Yes, File links are set relative to page in RW settings. Same project worked fine in S2.7.

**Comment by yourhead** [comment]
First off:  paths are complex. i would not be surprised if there is a bug here -- however, i have not been able to duplicate the bug that you've mentioned.

I compared S2 vs. S3 for this value and cannot find any differences in my test pages.

I looked through code history and the only changes I found are in error-states.  The values returned when the RW API returns garbage were changed. However normal condition values have not been changed that I could find.  I would not expect the error states to have made a difference (although it's not out of the question).

Second:  i've open sourced my basic test stack that shows all of the path values along with their expect results. you can use this stack in both S2 and S3 and all values should return identical results except for baseURL which is a new S3 feature.
you can find it here:
https://github.com/yourhead/PathsTestStack

Details about paths:  all path templates have 3 possible values.  one each for Edit, Preview, and Publish mode.  Although some are the same across modes, most are not.  Furthermore, Publish mode is affected by the user's settings in the RapidWeaver site Advanced Settings for "Paths are relative to: ..."

In other words:  There are a lot of possible values.

Both "resources" and "../resources" are expected values in at least one mode.  So your statement above is both true and false:  Those are both both returned (in at least one mode) by S2.7 -- but they are also both returned by S3 (in the same mode as S2).

I'd encourage you to look at my test stack and compare it with your own.  If we can find where things differ perhaps we can get at the source of your bug.

**Comment by jrondorf** [comment]
http://test.instacks.com/pathtest/page1/

I guess there is the bug: [path to docRoot] is not added to resources 

relativeDocRoot
Path: %relativeDocRoot% 
Expected (Export): [path to docroot] 
Actual: ../ 

rapidweaverAssetPath
Path: %rapidweaverAssetPath% 
**Expected (Export): [path to docRoot]resources**
**Actual: resources**

**Comment by yourhead** [comment]
yes, however this is the behavior you'll find in all versions of stacks 2. it's a bug that has existed for so long that it has become the expected behavior.  :-(

**Comment by yourhead** [comment]
i was so certain i had looked at that. but now it seems quite obvious that i overlooked something. thanks for double checking.

**Comment by jrondorf** [comment]
That is the (correct) outcome in S271: http://test.instacks.com/pathtestS271/page1/

relativeDocRoot
Path: %relativeDocRoot% 
Expected (Export): [path to docroot] 
Actual: ../ 

rapidweaverAssetPath
Path: %rapidweaverAssetPath% 
**Expected (Export): [path to docRoot]resources**
**Actual: ../resources**
